### PR TITLE
don't call M<K><n> releases the latest

### PR DESCRIPTION
**Pull request by aryairani**
...though this will exclude anything with an `M` in it, which may be wrong someday.

### CODE REVIEW

**Review by ChrisPenner — COMMENTED** [review]
Hrmm, not sure this'll work as is; e.g. if you run this:

```
1.10.0
1.5.0
```

Through `sort -r` it puts `1.5.0` as the highest version because it sorts lexicographically. Normally `-n` solves that, but I don't think it works with the dots in there :'(

**Comment by aryairani** [comment]
@ChrisPenner Ack, ok thanks. Though I think the code that doesn't work is unfortunately already in, not affected by this PR, yeah?

**Comment by ChrisPenner** [comment]
The code is there yup, it worked okay for small-number M-style releases, but will be problematic with semver if we hit two-digit version numbers 😬

**Comment by ChrisPenner** [comment]
EDIT: Apparently `sort -rV` does the thing we want as long as we don't use release-candidates, which I think we've been avoiding anyways so far, maybe just swap it to use that :) 

https://stackoverflow.com/a/40391207

File: scripts/make-release.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ usage() {
     echo "E.g."
     echo "$0 M4a"
     echo ""
-    echo "The latest release is: $(git tag --list 'release/*' | sort -r | head -n 1 | sed 's/release\///')"
+    echo "The latest release is: $(git tag --list 'release/*' | grep -v M | sort -r | head -n 1 | sed 's/release\///')"
```
Comment (ChrisPenner) [inline scripts/make-release.sh]:
```suggestion
    echo "The latest release is: $(git tag --list 'release/*' | grep -v M | sort -rV | head -n 1 | sed 's/release\///')"
```

**Review by ChrisPenner — APPROVED** [review]
Left a suggestion that should fix the sort ordering issue :)

**Comment by aryairani** [comment]
Thank you @ChrisPenner

File: scripts/make-release.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ usage() {
     echo "E.g."
     echo "$0 M4a"
     echo ""
-    echo "The latest release is: $(git tag --list 'release/*' | sort -r | head -n 1 | sed 's/release\///')"
+    echo "The latest release is: $(git tag --list 'release/*' | grep -v M | sort -rV | head -n 1 | sed 's/release\///')"
```
Comment (aryairani) [inline scripts/make-release.sh]:
```suggestion
    echo "I think the latest release is: $(git tag --list 'release/*' | grep -v M | sort -rV | head -n 1 | sed 's/release\///')"
```

**Pull request merged**